README.md

```diff
@@ -427,14 +427,16 @@ In order to enable the plugin you need to add the table of contents button to th
 ```
 ### Configuring the plugin with a custom config
 
-You can configure the nodeview with the hiearchy of the nodes
+You can configure the nodeview with the hiearchy of the nodes, by default clicking on a link in the table of contents it will put that element in the bottom of the page. You can make it work so it appears on top if you specify a function that returns the container that contains the scrolling for the document.
```

Comment:
with the new functionality, isn't it possible to avoid having to pass the scrollcontainer and get it directly in the app instead?

Reply:
I did it, but I'm not 100% sure it will work for all the cases, right now at least it works for all the cases we use, so I left the option to specify a scrollContainer in case they need to do so